content/en/docs/concepts/traffic-management/index.md

```diff
@@ -1252,75 +791,120 @@ spec:
         subset: v1
 {{< /text >}}
 
-### Return an HTTP 400 error code for 10% of requests
+For detailed instructions on how to configure delays and aborts, see
+[Fault Injection](/docs/tasks/traffic-management/fault-injection/).
+
+### Working with your applications {#working-with-your-applications}
+
+-   Istio failure recovery features are completely transparent to the
```

Comment:
One more nit: why are these bullets instead of paragraphs?

Reply:
Good point. I think they were small bullets when I started and then grew....

I'll de-bullet them.